src/Nox.Generator/Application/Dto/ReferencesDto.template.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Generated
+
+#nullable enable
+using Nox.Types;
+using Microsoft.EntityFrameworkCore;
+
+namespace {{codeGeneratorState.ApplicationNameSpace}}.Dto;
+
+public partial class ReferencesDto<T>
```

Comment:
```suggestion
public partial class {{className}}<T>
```

Reply:
Moved this class from template to Nox.Lib